File: TOMIMSV4/Entity/Mantenimientos/Producto/Producto_Talla/clsBeTalla.vb
Public Class clsBeTalla
    Implements ICloneable

    Public Property IdTalla() As Integer = 0
    Public Property Nombre() As String = ""
    Public Property Descripcion() As String = ""
    Public Property IdPropietario() As Integer = 0
    Public Property Fec_agr() As Date = Now
    Public Property User_agr() As String = ""
    Public Property Fec_mod() As Date = Now
    Public Property User_mod() As String = ""
    Public Property Activo As Boolean = False
    Public Property Codigo As String = ""
    Public Property IsNew As Boolean = False

    Sub New()
        '#EJC: Add your constructor here... 
    End Sub

    Public Function Clone() As Object Implements System.ICloneable.Clone
        Return MyBase.MemberwiseClone()
    End Function

End Class

File: WMS.EntityCore/Datos_Maestros/clsBeTalla.cs
using System;
using System.ComponentModel;
using System.ComponentModel.DataAnnotations.Schema;

namespace WMSWebAPI.Be
{
    public class clsBeTalla : ICloneable
    {
        [Column("IdTalla")]
        [DisplayName("IdTalla")]
        public int IdTalla { get; set; } = 0;

        [Column("Nombre")]
        [DisplayName("Nombre")]
        public string Nombre { get; set; } = "";

        [Column("Descripcion")]
        [DisplayName("Descripcion")]
        public string Descripcion { get; set; } = "";

        [Column("IdPropietario")]
        [DisplayName("IdPropietario")]
        public int IdPropietario { get; set; } = 0;

        [Column("fec_agr")]
        [DisplayName("fec_agr")]
        public DateTime Fec_agr { get; set; } = DateTime.Now;

        [Column("user_agr")]
        [DisplayName("user_agr")]
        public string User_agr { get; set; } = "";

        [Column("fec_mod")]
        [DisplayName("fec_mod")]
        public DateTime Fec_mod { get; set; } = DateTime.Now;

        [Column("user_mod")]
        [DisplayName("user_mod")]
        public string User_mod { get; set; } = "";

        [Column("activo")]
        [DisplayName("activo")]
        public bool Activo { get; set; } = false;

        public clsBeTalla() { }

        public object Clone()
        {
            return this.MemberwiseClone();
        }
    }
}
